conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -13,9 +13,13 @@
 
 
 class AutotoolsToolchain:
+    def __new__(cls, conanfile, *args, **kwargs):
+        if conanfile.use_future and "AutotoolsToolchain" in conanfile.use_future:
```

Comment:
I started using a `conf`: https://github.com/conan-io/conan/pull/16010/commits/ac267694122f9d4286ae9e855bb5744f961bcf83 But I thought that it was not very practical when using upstream recipes, so that's why I decided to use a new ConanFile property.

Reply:
I like this approach better too, even if it smells a bit iffy :)